src/styles/constants/_layout.scss

```diff
@@ -21,3 +21,7 @@ $bdl-grid-unit-px: 4 !default; // unitless grid unit (defined in pixels for manu
 
 // Headers
 $bdl-header-height: 64px;
+
+// Message
+$message-line-clamp: 7;
+$message-max-height: $message-line-clamp * 20px; // line-height for message is 20px with font-size of 13px, mostly for IE11
```

Comment:
I understand we created these variables to share them between the component/stylesheet. However, these are global values. Do we intend to use them elsewhere? What happens if other experiences have different font sizes and/or line heights? It may be preferable to move the values into the JS component and use them via inline styles. What do you think?

Reply:
I need some time to think this through. I don't like to use inline styles.

Reply:
I would also hesitate to put them in this file/directory. at minimum they would need a `bdl-` prefix.

since they only have one usage at the moment, is it okay to have them as local variables in `ActivityMessage.scss`?

Reply:
I wanted them in `ActivityMessage.scss` but the `:export {}` syntax didn't worked. Then I found that we have build script for generating js files with sass variables.
I think I'll go with `styles`.